setup.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 set -ex
 
+PYTHON=''
+INSTALLFLAGS=''
+
+if [ "$1" == "--user" ] || (uname | grep -i Darwin &>/dev/null); then
```

Comment:
```bash
(uname | grep -i Darwin &>/dev/null)
```
Is this for checking whether we are on a Mac/OSX?

If so, maybe move it to a function:
```bash
is_osx() {
  uname | grep -i Darwin &>/dev/null
}
```